dylint/src/lib.rs

```diff
@@ -306,57 +306,6 @@ fn name_as_path(name: &str, as_path_only: bool) -> Result<Option<(String, PathBu
     Ok(None)
 }
 
-fn list_lints(opts: &opts::Dylint, resolved: &ToolchainMap) -> Result<()> {
```

Comment:
Could this change be unnecessary? I see `list_lints` called here: https://github.com/trailofbits/dylint/blob/1248293e722d4017ef59821dd0eb3dca28f328d8/dylint/src/lib.rs#L139

Reply:
I see. `check_or_fix` is called on the line before the call to `list_lints`.
```
warning: 
   --> dylint/src/lib.rs:360:1
    |
360 | / fn check_or_fix(
361 | |     opts: &opts::Dylint,
362 | |     check_opts: &opts::Check,
363 | |     resolved: &ToolchainMap,
...   |
470 | | }
    | |_^ function `check_or_fix` should be defined before `list_lints`
    |
    = help: move check_or_fix's definition to earlier in the module
    = note: `#[warn(non_topologically_sorted_functions)]` on by default
```
I withdraw this question.